Sources/VLCTabBarCoordinator.swift

```diff
@@ -65,3 +65,9 @@ extension UITabBarController {
         return PresentationTheme.current.colors.statusBarStyle
     }
 }
+
+extension UINavigationController {
```

Comment:
Shouldn't we put it in the Appearance manager to the other extension?

Reply:
I'm not sure to understand, would you prefer me to put in the Appearance manager?

Reply:
yes we have already an extension for statusbar there as well :) 
```/extensions so that preferredStatusBarStyle is called on all childViewControllers otherwise this is not forwarded
extension UINavigationController {
    override open var preferredStatusBarStyle: UIStatusBarStyle {
        return PresentationTheme.current.colors.statusBarStyle
    }
}
```

Reply:
and I would rather search there than in the tabbarcoordinator

Reply:
Ah, I get it now! Thank you for the clarification, I will make the changes.